polkadot-parachains/integritee-runtime/src/lib.rs

```diff
@@ -699,6 +698,7 @@ construct_runtime! {
 		RandomnessCollectiveFlip: pallet_randomness_collective_flip::{Pallet, Storage} = 2,
 		Timestamp: pallet_timestamp::{Pallet, Call, Storage, Inherent} = 3,
 		ParachainInfo: parachain_info::{Pallet, Storage, Config} = 4,
+		Preimage: pallet_preimage = 5,
```

Comment:
Todo: double check if we have never used index 5 to be sure that we don't have a storage hash collision.